docs/http.md

```diff
@@ -437,30 +437,42 @@ step.
   | geojson    | [GeoJSON `LineString`](http://geojson.org/geojson-spec.html#linestring) or [GeoJSON `Point`](http://geojson.org/geojson-spec.html#point) if it is only one coordinate (not wrapped by a GeoJSON feature)|
   
 - `name`: The name of the way along which travel proceeds.
+- `lanes`: the available turn lanes at the turn
```

Comment:
This is in fact a documentation mix-up. They are part of `StepManeuver` in the code already.